CONTRIBUTING.md

```diff
@@ -91,6 +91,7 @@ effect on highlighting. We will work on improving highlighting in the near futur
 
 ```
 @comment
+@spell
```

Comment:
Hmm, I was thinking to add it as a separate section. In any case, it would be good to mention the special purpose.